wrappers/obj-c/ODWLogConfiguration.mm

```diff
@@ -4,16 +4,341 @@
 //
 #import <Foundation/Foundation.h>
 #import "ODWLogConfiguration.h"
+#import "ODWLogConfiguration_private.h"
 #import "ODWLogger_private.h"
 #import "LogManager.hpp"
 
 using namespace Microsoft::Applications::Events;
 
+/*!
+  Enable analytics.
+*/
+NSString *const ODWCFG_BOOL_ENABLE_ANALYTICS = @"enableLifecycleSession";
```

Comment:
Why did you not define these at ODWLogConfiguration.h directly?

Reply:
Isn't that how public static vars are defined in objc? https://stackoverflow.com/a/23653584/4092412 https://stackoverflow.com/a/3117562/4092412 https://stackoverflow.com/a/16331068/4092412